doc/dev/index.rst

```diff
@@ -34,7 +34,7 @@ notice that a link is broken or if you know of a better link, please
 
 .. _`report it as a bug`: http://tracker.ceph.com/projects/ceph/issues/new
 
-Essentials (tl;dr)
+Essentials
```

Comment:
Aha, I thought they are just some gibberish. Thank you for your explanation, will drop it.